src/atomate2/common/files.py

```diff
@@ -35,11 +35,11 @@ def copy_files(
         either "usernamehost" or just "host" in which case the username
         will be inferred from the current user. If ``None``, the local filesystem will
         be used as the source.
-    include_files : None or list of (str or .Path)
+    include_files : None | list[str | Path]
```

Comment:
Is there a reason you changed the doc strings? I think the correct style is to use "or" in the string since it is human readable. E.g. see here https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
I didn't make these changes, they're from [here](https://github.com/materialsproject/atomate2/pull/854/commits/fbb94a42b4c9a16634056954db34d8d5e343a0ae). But I'll revert them

Reply:
OK reverted!